### ISSUE TITLE
sqlite storage + file plugins issue

### ISSUE THREAD

**Issue by rorp24** [brody2consult/cordova-sqlite-storage#872]
Hi. I want to get a database from a local file on my device. here is the code i use:
`this.sqlite.create({
			name: myfilename,
			location: this.file.externalDataDirectory+"mydbdirectory/"
		}).then((res: SQLiteObject) =>{
			/*do something*/
		}, err =>{
			console.error("sql error:" + JSON.stringify(err))
			/*do something else*/
		})`

But it go to "err", and it just read me this: "sql error:{"code":0}"

I tried to ask to our lord and savior internet, but it didn't help me. Even stackoverflow didn't tell me the solution.

So i ask to the only person who can help me: you. Can you tell me what's wrong, and, if you can, tell me how to solve my problem?

Thank you

**Comment by brody2consult** [comment]
There are multiple problems with this issue report:

- It looks like you are using some kind of third-party library, maybe Ionic, which is not *directly* supported here. The information about *which* library you are using is missing here.
- Assuming that the third-party library uses the given `name` and `location` values in an `openDatabase` call, the `location` value you are using is not supported by this plugin. This information is already documented.

**Comment by rorp24** [comment]
yes, i use ionic, and the link in the doc redirect on this repo. That why I was calling you to help. If you can't help me, maybe you know who can

**Comment by JackZhai2018** [comment]
the sqlite plugin is not friendly. this problem can not solved with document or forum. no one can give the solution to this problem. even the maintainer ?

**Comment by brody2consult** [comment]
Guys please do me and other project maintainers a favor. Critical feedback is always welcome but please keep it objective and more respectful in the future. Benefit of the doubt can sometime be extremely helpful in finding solutions that can work for everyone involved.

I am reopening this issue as a discussion for now. I will need some time to consider my thoughts on this issue.
